sdk/documentintelligence/azure-ai-documentintelligence/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,199 @@
+
+Service follows the GFM spec ([GitHub Flavored Markdown](https://github.github.com/gfm/)) for the Markdown format. This
+SDK introduces a new enum _ContentFormat_ with value "text" or "markdown" to indicate the result content format.
+
+```java
+File document = new File("{your-file-to-analyze}");
+SyncPoller<AnalyzeResultOperation, AnalyzeResultOperation> analyzeLayoutResultPoller =
+        client.beginAnalyzeDocument("prebuilt-layout", null,
+                null,
+                null,
+                null,
+                null,
+                ContentFormat.MARKDOWN,
+                new AnalyzeDocumentRequest().setBase64Source(Files.readAllBytes(document.toPath())));
+```
+For the complete sample, see [Sample: Markdown][Sample-AnalyzeLayoutMarkdownOutput].
+
+### Query Fields
+
+We reintroduce query fields as a premium add-on feature. When the keyword argument `features` is specified, the service
+will further extract the values of the fields specified via the keyword argument `queryFields` to supplement any 
+existing fields defined by the model as fallback.
+
+```java
+File document = new File("{your-file-to-analyze}");
+SyncPoller<AnalyzeResultOperation, AnalyzeResultOperation> analyzeLayoutResultPoller =
+        client.beginAnalyzeDocument("prebuilt-layout", null,
+                null,
+                null,
+                Arrays.asList(DocumentAnalysisFeature.QUERY_FIELDS),
+                Arrays.asList("Address", "InvoiceNumber"),
+                null,
+                new AnalyzeDocumentRequest().setBase64Source(Files.readAllBytes(document.toPath())));
+```
+For the complete sample, see [Sample: Query Fields][Sample-AnalyzeAddOnQueryFields].
+
+### Split Options
+
+In the previous API versions supported by the older `azure-ai-formrecognizer` library, document splitting and
+classification operation (`"/documentClassifiers/{classifierId}:analyze"`) always tried to split the input file into
+multiple documents.
+
+To enable a wider set of scenarios, this SDK introduces a keyword argument "split" to specify the document splitting 
+mode with the new "2023-10-31-preview" service API version. The following values are supported:
+
+- `split: "auto"`
+
+Let service determine where to split.
+
+- `split: "none"`
+
+The entire file is treated as a single document. No splitting is performed.
+
+- `split: "perPage"`
+
+Each page is treated as a separate document. Each empty page is kept as its own document.
+
+
+## Breaking Changes
+
+### Clients names updates
+To improve the development experience and address the consistent feedback across the Document Intelligence SDK, this new
+version of the library introduces two new clients
+`DocumentIntelligenceClient` and the `DocumentIntelligenceAdministrationClient` that provide unified methods for
+analyzing documents and provide support for the new features added by the service in
+API version `2023-10-31-preview` and later.
+
```

Comment:
nit: Can you have a table to show the relationship of SDK and API version which would more straightforward?